src/components/Reports/Shared/CalculationReports/DirectionButtons/DirectionButtons.tsx

```diff
@@ -60,7 +63,23 @@ export const DirectionButtons: React.FC<DirectionButtonsProps> = ({
   };
 
   return (
-    <Box sx={{ mt: 5, display: 'flex', justifyContent: 'flex-end' }}>
+    <Box
+      sx={{
+        mt: 5,
+        display: 'flex',
+        justifyContent: showCancelButton ? 'space-between' : 'flex-end',
+      }}
+    >
+      {showCancelButton && (
```

Comment:
What if we only show the cancel button if `handleCancel` is defined? It would save us a parameter